sdk/eventhub/ci.yml

```diff
@@ -1,25 +1,38 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
 trigger:
   branches:
     include:
       - master
       - feature/*
+      - release/*
+      - hotfix/*
   paths:
     include:
-      - sdk/eventhub/
+      - sdk/eventhubs/
```

Comment:
The path is currently `sdk/eventhub`. I know some folks want to change that but until that happens you need to remove the s :)

Reply:
Fixed.